go/ephemeral/team_ek.go

```diff
@@ -367,7 +367,7 @@ func fetchTeamMemberStatements(ctx context.Context, g *libkb.GlobalContext, team
 		// Verify the server only returns actual members of our team.
 		uv, err := team.UserVersionByUID(ctx, uid)
 		if err != nil {
-			return nil, err
+			continue // user is not on the team chain
```

Comment:
@mlsteele what is the correct way to check for membership in a team? the server was accidentally sending down the uid of a reset user and the `team.IsMember(ctx, uv)` call below succeeded

Reply:
discussed offline, it's because the UV is a member of the team, we need to load the latest eldestseqno to verify membership